### PR TITLE
fix: blocked user UI fixes

### PR DESCRIPTION
**Pull request by borichellow**
# What's new in this PR?

### Issues

1. in OtherUserProfileScreen: for blocked user OptionsBottomSheet was displayed as empty
2. in ConversationsListScreen: BlockUserDialog do not disappear after blocking user  
3. in ConversationScreen: with blocked user message editText was not disabeled (sometimes)
4. Block user not working when just recently connected

### Causes (Optional)

1. the bottomSheet content was not specified for the blocked user 🤷‍♂️

2. BlockUserDialog disappears once request is done, for that we have `MutableState requestInProgress` in viewModel. So it is set to true once use click the button and suppose to set to false when request is done. But for some reason setting to false was done outside of the CoroutineScope where request happens. So it was like: set to true and set to false almost at the same time.

3. the problem was that `isUserBlocked` was checked from the UiState and should be from the data that came from DB 🤷‍♂️

4. fixed by kalium update

### Dependencies (Optional)

Depends on kalium PR

Needs releases after:

https://github.com/wireapp/kalium/pull/998

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-1056/3/) **failed**.

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-1056/4/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4551311)(4551311)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4551311)(4551311)-dev-release.apk)

**Comment by AndroidBob** [comment]
Build [5](https://10.10.124.134/job/wire-android-reloaded/job/PR-1056/5/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4581378)(4581378)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4581378)(4581378)-dev-release.apk)

File: app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserProfileScreenViewModel.kt [hunk 1]
```diff
@@ -309,6 +313,7 @@ class OtherUserProfileScreenViewModel @Inject constructor(
     }
 
     private suspend fun showInfoMessage(type: SnackBarMessage) {
+        _closeBottomSheet.emit(Unit)
```
Comment (saleniuk) [inline app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserProfileScreenViewModel.kt:316]: Should we always close the bottom sheet when showing the snackbar? If so maybe the name of this function should be updated to know that it doesn't just show the info message 😉

**Review by saleniuk — APPROVED** [review]
Generally looks ok, just one little comment 😄

File: app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserProfileScreenViewModel.kt (hunk 1 above)
Comment (borichellow) [reply app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserProfileScreenViewModel.kt:316]: make sense :)

**Comment by AndroidBob** [comment]
Build [6](https://10.10.124.134/job/wire-android-reloaded/job/PR-1056/6/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4582475)(4582475)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4582475)(4582475)-dev-release.apk)

**Comment by AndroidBob** [comment]
Build [7](https://10.10.124.134/job/wire-android-reloaded/job/PR-1056/7/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4582611)(4582611)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4582611)(4582611)-dev-release.apk)

**Comment by AndroidBob** [comment]
Build [8](https://10.10.124.134/job/wire-android-reloaded/job/PR-1056/8/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4582778)(4582778)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4582778)(4582778)-dev-release.apk)

**Pull request merged**